src/Web/Avalonia.Web.Blazor/Cursor.cs

```diff
@@ -0,0 +1,93 @@
+using Avalonia.Input;
+using Avalonia.Platform;
+
+namespace Avalonia.Web.Blazor
+{
+    public class CssCursor : ICursorImpl
```

Comment:
Note: this class is public, not internal, to allow end users to use some custom constructors that will allow to use cursors that's is not in the `StandardCursorType` enum values list